sim/midas/src/main/cc/simif_emul_vcs.cc

```diff
@@ -37,6 +37,13 @@ void simif_emul_vcs_t::sim_init() {
   for (size_t i = 0; i < 10; i++)
     target.switch_to();
   vcs_rst = false;
+
```

Comment:
This initialisation happens after target reset, but before INIT_DONE is asserted. I think this should be moved into an initialisation callback called from the simulation main loop after waiting for INIT_DONE to be true.